pkg/nvml/const.go

```diff
@@ -78,6 +78,24 @@ const (
 	VGPU_NAME_BUFFER_SIZE = 64
 	// GRID_LICENSE_FEATURE_MAX_COUNT as defined in nvml/nvml.h
 	GRID_LICENSE_FEATURE_MAX_COUNT = 3
+	// VGPU_SCHEDULER_POLICY_UNKNOWN as defined in nvml/nvml.h
+	VGPU_SCHEDULER_POLICY_UNKNOWN = 0
```

Comment:
OK. I see, they re-added them here.

Reply:
Yeah, they were removed in 12.1 for some reason and then readded in 12.2